namespaces/cloud-platform-live-0.k8s.integration.dsd.io/categorisation-tool-dev/resources/pathfinder_s3.tf

```diff
@@ -0,0 +1,34 @@
+terraform {
+  backend "s3" {}
+}
+
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "pathfinder_offender_data_omic_s3_bucket" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=1.0"
+
+  team_name              = "cloudplatform"
+  business-unit          = "mojdigital"
+  application            = "cloud-platform-terraform-s3-bucket"
+  is-production          = "false"
+  environment-name       = "development"
+  infrastructure-support = "platform@digtal.justice.gov.uk"
+}
```

Comment:
Hi, can you change the tags to reflect your team name and application. This applies to all resources you've requested.

Reply:
Hi @jasonBirchall I've updated the s3 details.